README.md

```diff
@@ -1,2 +1,133 @@
+  - [Exporting the Ledger Chain](#exporting-the-ledger-chain)
+   - [Command Reference](#command-reference)
+		- [Bucket List Commands](#bucket-list-commands)
+	       - [export_accounts](#export_accounts)
+	       - [export_offers](#export_offers)
+	       - [export_trustlines](#export_trustlines)
+		- [History Archive Commands](#history-archive-commands)
+		   - [export_ledgers](#export_ledgers)
+		   - [export_transactions](#export_transactions)
+		   - [export_operations](#export_operations)
+		- [Stellar Core Commands](#stellar-core-commands)
+		   - [export_ledger_entry_changes](#export_ledger_entry_changes)
+    - [Schema](#schema)
+
+
+
+
+## Exporting the Ledger Chain
+
+1. Install Golang: https://golang.org/dl/
+
+2. Ensure that your Go bin has been added to the PATH env variable: `export PATH=$PATH:$(go env GOPATH)/bin`
+3. Download and install Stellar-Core v13.2.0 or later: https://github.com/stellar/stellar-core/blob/master/INSTALL.md
+
+4. Run `go get github.com/stellar/stellar-etl` to install the ETL
+
+5. Run export commands in order to export information about the legder
+
+## Command Reference
+- [Bucket List Commands](#bucket-list-commands)
+   - [export_accounts](#export_accounts)
+   - [export_offers](#export_offers)
+   - [export_trustlines](#export_trustlines)
+- [History Archive Commands](#history-archive-commands)
+   - [export_ledgers](#export_ledgers)
+   - [export_transactions](#export_transactions)
+   - [export_operations](#export_operations)
+ - [Stellar Core Commands](#stellar-core-commands)
+   - [export_ledger_entry_changes](#export_ledger_entry_changes)
+
+Every command accepts a `-h` parameter, which provides a help screen containing information about the command, its usage, and its flags.
+
+### Bucket List Commands
+
+These commands use the bucket list in order to ingest large amounts of data from the history of the stellar ledger. If you are trying to read large amounts of information in order to catch up to the current state of the ledger, these commands provide a good way to catchup quickly. However, they don't allow for custom start-ledger values. For updating within a user-defined range, see the Stellar Core commands.
+
+#### export_accounts
+
+```bash
+> stellar-etl export_accounts --end-ledger 500000 --output exported_accounts.txt
+```
+
+This command exports accounts, starting from the genesis ledger and ending at the ledger determined by `end-ledger`. This command exports the point in time state of accounts, meaning that the exported data represents the account information as it was at `end-ledger`.
+
+#### export_offers
+
+```bash
+> stellar-etl export_offers --end-ledger 500000 --output exported_offers.txt
+```
+
+This command exports offers, starting from the genesis ledger and ending at the ledger determined by `end-ledger`. This command exports the point in time state of offer, meaning that the exported data represents the offerbook as it was at `end-ledger`.
```

Comment:
Nit: orderbook, not offerbook, for consistency with other Stellar terminology. Would also say "point-in-time state of offers on Stellar's decentralized exchange," or something similar, just to be clear.